nowcasting_dataset/dataset/datasets.py

```diff
@@ -153,26 +191,7 @@ def __getitem__(self, batch_idx: int) -> example.Example:
             sat_datetime_index=netcdf_batch.sat_time_coords,
             nwp_target_time=netcdf_batch.nwp_time_coords,
         )
-        for key in [
-            "nwp",
-            "nwp_x_coords",
-            "nwp_y_coords",
-            "sat_data",
-            "sat_x_coords",
-            "sat_y_coords",
-            "pv_yield",
-            "pv_system_id",
-            "pv_system_row_number",
-            "pv_system_x_coords",
-            "pv_system_y_coords",
-            "x_meters_center",
-            "y_meters_center",
-            GSP_ID,
-            GSP_YIELD,
-            GSP_X_COORDS,
-            GSP_Y_COORDS,
-            GSP_DATETIME_INDEX,
-        ] + list(nowcasting_dataset.consts.DATETIME_FEATURE_NAMES):
+        for key in self.required_keys + list(nowcasting_dataset.consts.DATETIME_FEATURE_NAMES):
```

Comment:
Should we put the datetime feature into require_keys, all put a separate input as 'datetime_required_keys'

Reply:
I think it would make sense to put them in the required_keys, so that its all in one place.

Reply:
Added it to the required_keys